package com.exception;

import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;
 
public class Exception_Types {
	
	public static void main(String[] args) {
		
		
		
		/*
		 * int x = 25; int y = 0; System.out.println(x/y);
		 * 
		 * System.out.println("Done");
		 */
		  
		  
		/*
		 * List<Integer> L = new ArrayList<Integer>(); L.add(20);
		 * System.out.println(L.get(3));
		 */
		 
		
		/*
		 * String b = "Krish"; char charAT =b.charAt(5); System.out.println(charAT);
		 */
		
		/*
		 * int a[] = new int[3]; System.out.println(a[3]);
		 */
		/*
		 * Scanner S =new Scanner (System.in); System.out.println("Enter the number");
		 * int number = S.nextInt(); System.out.println(number);
		 * 
		 * String a = null; int length = a.length(); System.out.println(length);
		 */
		
		String a1 = "Krish123";
		int parseInt = Integer.parseInt(a1);
		System.out.println(parseInt);
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
	}

}
